tests/test_upload.py

```diff
@@ -72,6 +73,35 @@ def test_successs_prints_release_urls(upload_settings, stub_repository, capsys):
     assert captured.out.count(NEW_RELEASE_URL) == 1
 
 
+def test_print_packages_if_verbose(upload_settings, capsys):
+    """Print the path and file size of each distribution attempting to be uploaded."""
+    dists_to_upload = [
+        helpers.WHEEL_FIXTURE,
+        helpers.SDIST_FIXTURE,
+        helpers.NEW_SDIST_FIXTURE,
+        helpers.NEW_WHEEL_FIXTURE,
+    ]
+
+    expected_sizes_of_dists_to_upload = ["15.4 KB", "20.8 KB", "26.1 KB", "21.9 KB"]
+
+    upload_settings.verbose = True
+
+    result = upload.upload(upload_settings, dists_to_upload)
+
+    assert result is None
+
+    captured = capsys.readouterr()
+
+    for index in range(len(dists_to_upload)):
+        curr_dist = dists_to_upload[index]
+        size_of_curr_dist = utils.get_file_size(curr_dist)
```

Comment:
There are more Pythonic ways to loop indexing into a list. One would be to use a dictionary, e.g.:

```python
dists_to_upload = {
    helpers.WHEEL_FIXTURE: "15.4 KB",
    # ...
}

# ...

result = upload.upload(upload_settings, dists_to_upload.keys())

# ...

for filename, size in dists_to_upload.items():
```

Also, I think the call to `get_file_size` isn't necessary, since it's already unit-tested, and this is more of an integration test to make sure that the file size is printed.

Finally, instead of multiple assertions, I might have one that asserts that expected formatted string, i.e. `f"{filename} ({size})"`, is in the output.